src/UCTNode.cpp

```diff
@@ -207,10 +215,32 @@ void UCTNode::set_policy(float policy) {
     m_policy = policy;
 }
 
+float UCTNode::get_variance(float default_var) const {
+    return m_visits > 1 ? m_squared_diff / (m_visits - 1) : default_var;
+}
+
+float UCTNode::get_stddev(float default_stddev) const {
```

Comment:
get_eval_stddev()?

Reply:
Removed the `get_stddev` function instead since it wasn't used.